clients/drcachesim/drpt2trace/drpt2trace.cpp

```diff
@@ -74,9 +74,18 @@ static droption_t<std::string>
               "Specifies the file path of the PT raw trace. Please run the "
               "libipt/script/perf-read-aux.bash script to get PT raw trace file from the "
               "data generated by the perf record command.");
+static droption_t<std::string>
+    op_elf(DROPTION_SCOPE_FRONTEND, "elf", "", "[Optional] Path of the elf file",
+           "Specifies the file path of the elf file. You must specify when "
+           "converting short traces that don't contain sideband information.");
+static droption_t<unsigned long long>
+    op_elf_base(DROPTION_SCOPE_FRONTEND, "elf_base", 0x0,
+                "[Optional] The base address of the elf file",
```

Comment:
What base is this?  The preferred base hardcoded in the file, or the runtime base at which the file was loaded during tracing?  Please clarify in these docs.  I assume the runtime base since the preferred base is available from the file and should not need to be passed in.  But if it's the runtime base, why is it optional?  Do you assume it's loaded at its preferred base otherwise?  If it's PIC/PIE the preferred base will nearly always be 0 so this would be required there.